### PR TITLE
BaSP-TG-158: implement soft delete

### PR DESCRIPTION
**Pull request by ProDIGGY-vinilos**

### CODE REVIEW

File: src/Components/Pages/Admin/Projects/ProjectList/index.js [hunk 1]
```diff
@@ -1,13 +1,17 @@
 import Table from 'Components/Shared/Table';
 import React, { useEffect, useState } from 'react';
-import { getProjectsByEmployee, deleteProject } from 'redux/projects/thunks';
+import {
+  getProjects,
+  // getProjectsByEmployee,
```
Comment (fede230298) [inline src/Components/Pages/Admin/Projects/ProjectList/index.js:5]: remove comment

File: src/Components/Pages/Admin/Projects/ProjectList/index.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+  deleteProject
+} from 'redux/projects/thunks';
 import { useDispatch, useSelector } from 'react-redux';
 import MessageModal from 'Components/Shared/Modal/MessageModal';
 import Button from 'Components/Shared/Button/Button';
 import styles from 'Components/Pages/Admin/Projects/ProjectList/adminProjectList.module.css';
 
 const ProjectsPage = () => {
-  const userId = '636c1e8ddabe537336ae082a';
+  // const userId = '636c1e8ddabe537336ae082a';
```
Comment (fede230298) [inline src/Components/Pages/Admin/Projects/ProjectList/index.js:14]: remove comment

File: src/Components/Pages/Admin/Projects/ProjectList/index.js (hunk 1 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Pages/Admin/Projects/ProjectList/index.js]: done

File: src/Components/Pages/Admin/Projects/ProjectList/index.js (hunk 2 above)
Comment (ProDIGGY-vinilos) [reply src/Components/Pages/Admin/Projects/ProjectList/index.js]: done

File: src/Components/Pages/Admin/Projects/ProjectForm/index.js [hunk 3]
```diff
@@ -30,7 +30,7 @@ const Project = () => {
   const [showSharedModal, setShowSharedModal] = useState(false);
 
   const { isLoading, error } = useSelector((state) => state.projects);
-  const { list: employeesList } = useSelector((state) => state.employees);
+  const { list: employeesListt } = useSelector((state) => state.employees);
```
Comment (JuliMonte5) [inline src/Components/Pages/Admin/Projects/ProjectForm/index.js:33]: employeesList* delete extra t

Comment (ProDIGGY-vinilos) [reply src/Components/Pages/Admin/Projects/ProjectForm/index.js:33]: Done

**Review by GonzaloAsencio — APPROVED** [review]

**Review by JuliMonte5 — APPROVED** [review]

**Pull request merged**